### PR TITLE
Unify section spacing

### PR DESCRIPTION
**Pull request by chasulin**
Sections on the location page had inconsistent spacing--this unifies the spacing by removing top/bottom margin from inner elements (headers, footers, inner containers) and places it on the shared `LocationBlock` component. Also cleans up homepage spacing a little bit for compare+explore

### CODE REVIEW

File: src/components/Compare/Compare.style.tsx [hunk 1]
```diff
@@ -45,20 +45,20 @@ export const Wrapper = styled.div<{
   max-height: ${({ $isModal }) => $isModal && '100vh'};
 
   @media (min-width: 600px) {
-    margin: ${({ $isModal }) => ($isModal ? '0 auto' : '3rem auto')};
+    margin: ${({ $isModal }) => ($isModal ? '0 auto' : '0 auto')};
```
Comment (chasulin) [inline src/components/Compare/Compare.style.tsx:48]: removes top and bottom margin from compare wrapper

File: src/assets/theme/sizes.tsx [hunk 2]
```diff
@@ -1,12 +1,9 @@
...
 // iPhone 5 screen is 320px wide. This breakpoint is used for iphone5 specific adjustments:
 export const smallPhoneBreakpoint = '321px';
 
-export default {
-  mobileBreakpoint,
-  materialSMBreakpoint,
-  smallPhoneBreakpoint,
-};
+// Breakpoint at which the county map changes to position:fixed on the location page:
+export const countyMapToFixedBreakpoint = '1320px';
```
Comment (chasulin) [inline src/assets/theme/sizes.tsx:9]: moved here from `CountyMap.tsx`

File: src/components/VaccinationEligibilityBlock/VaccinationEligibilityBlock.tsx [hunk 3]
```diff
@@ -76,7 +76,7 @@ const VaccinationEligibilityBlock: React.FC<{ region: Region }> = ({
 
   return (
     <Container>
-      <Heading2>Vaccine eligibility</Heading2>
+      <Heading2 style={{ marginTop: 0 }}>Vaccine eligibility</Heading2>
```
Comment (chasulin) [inline src/components/VaccinationEligibilityBlock/VaccinationEligibilityBlock.tsx:79]: putting this style inline for now. going to unify all of the section headings in a separate pr

**Review by pnavarrc — APPROVED** [review]
Discussed some remaining spacing issues on Slack, everything else looks good, thanks for tackling this! 👏 👏 👏 👏 👏

File: src/components/Compare/Compare.style.tsx [hunk 4]
```diff
@@ -45,20 +45,20 @@ export const Wrapper = styled.div<{
...
   }
 
   @media (min-width: 1350px) {
-    margin: ${({ $isModal }) => ($isModal ? '0 auto' : '3rem 445px 2rem auto')};
+    margin: ${({ $isModal }) => ($isModal ? '0 auto' : '0 445px 0 auto')};
   }
 
   @media (min-width: 1750px) {
-    margin: ${({ $isModal }) => ($isModal ? '0 auto' : '3rem auto 2rem')};
+    margin: ${({ $isModal }) => ($isModal ? '0 auto' : '0 auto')};
```
[the remark this answers is not in the document]
Comment (chasulin) [reply src/components/Compare/Compare.style.tsx:61]: oh yes good call

**Pull request merged**